projects/29_dynamic_rendering/DynamicRendering.cpp

```diff
@@ -139,6 +138,11 @@ void DynamicRenderingApp::Setup()
 
             preRecordedCmd->BeginRendering(&renderingInfo);
             {
+                // Clear RTV to greyish blue
+                grfx::RenderTargetClearValue rtvClearValue = {0.23f, 0.23f, 0.33f, 0};
```

Comment:
nit: any reasons to change the output?
(Internal tests should be fine if you do, but if changing the output is not required, might want to avoid)

```suggestion
                grfx::RenderTargetClearValue rtvClearValue = {0.7f, 0.7f, 0.7f, 1.f};
```

Reply:
changed to suggestion